scripts/harmonize.py

```diff
@@ -134,10 +134,13 @@ def __str__(self):
         cols.extend([format_num(self.gnomad_af, 3), format_num(self.af_fc, 3), self.gnomad_filt])
         return '\t'.join(cols)
 
-def harmonize(file_in, file_ref, chr_col, pos_col, ref_col, alt_col, af_col, beta_col, require_gnomad, passing_only, gnomad_min_an):
+def harmonize(file_in, file_ref, chr_col, pos_col, ref_col, alt_col, af_col, beta_col, require_gnomad, passing_only, gnomad_min_an, gnomad_max_abs_diff):
 
     if gnomad_min_an is None:
         gnomad_min_an = -1
+    
+    if gnomad_max_abs_diff is None:
+        gnomad_max_abs_diff = 1
```

Comment:
below when checking if ref_has_lines. check directly if the read returned empty string. 
Thats not exceptional situation that file ended. However if there are illegal values in the columns that is more exceptional or that there are wrong number of items. we control the file so should not happen.

Reply:
always better to check assumptions and error out though

Reply:
@juhaa remember to fix this also... don't catch as exception the end of file but check for empty return. Illegal values should raise exception and not be caught. Or you can catch them to log and then error out.

Reply:
Yup it's fixes. Haven't pushed the edits yet though.

Reply:
Resolved in https://github.com/FINNGEN/META_ANALYSIS/pull/46/commits/1108c713f6ad84656a405069195edd54025145ec.